cmd/bootstrap/main.go

```diff
@@ -19,7 +20,7 @@ func main() {
 	clusterType := os.Getenv("CLUSTER_TYPE")
 	switch clusterType {
 	case "group-replication":
-		if err := bootstrapGroupReplication(); err != nil {
+		if err := bootstrapGroupReplicationMySQLShell(context.TODO()); err != nil {
```

Comment:
You're in main here, so you can have `context.Background`

Reply:
https://github.com/percona/percona-server-mysql-operator/pull/316/commits/841e7ebea593d25ff2fca233f1b16df757ddfb55